conf/test_cram.config

```diff
@@ -4,19 +4,13 @@
  * -------------------------------------------------
  * Defines bundled input files and everything required
  * to run a fast and simple test. Use as follows:
- *     nextflow run qbic-pipelines/bamtofastq -profile test_cram
+ *     nextflow run qbic-pipelines/bamtofastq -profile test_cram,<docker/singularity> --outdir <OUTDIR>
  */
 
+includeConfig 'test.config'
```

Comment:
Can we avoid `includeConfig` cascade?
I was rather fond of it, but I know it can get messy, all `includeConfig` should be done only in `nextflow.config`

Reply:
Sorry, I was not aware of this. Will change it